### DISCUSSION TITLE
Levar Big Kahunas da Python Norte 2024 à PyBr24

### DISCUSSION THREAD

**Discussion by meu-bem** [apyb/comunidade#151] [Geral]
Durante uma conversa com a @talitarossari e a @anadulce na Python Norte, observamos que este é o primeiro ano, desde a pandemia, em que todas as edições regionais dos eventos de Python estão ocorrendo no Brasil. Em vista disso, e com o objetivo de fortalecer a integração entre os eventos regionais e aumentar a representatividade da Python Norte junto aos demais, gostaríamos de reunir os Big Kahunas de cada região durante a Python Brasil 2024.

Entretanto, tanto a Dâmaris quanto eu, como Kahunas da Python Norte 2024, não temos condições de custear nossa participação no evento por conta própria. Conversamos com a organização da Python Norte deste ano e como entendemos que cada evento tem autonomia de diante dos seus gastos acordamos internamente de utilizar o saldo da Python Norte de 2024 para custar o custo de viagem e estadia durante o evento, representando assim a comunidade Python Norte no evento e aproximando ainda mais as comunidades do Brasil.
Mas gostaríamos de validar com a APyB se há algum motivo para não seguimos com o planejamento.

**Comment by tiidadavena** [comment]
No tocante da demanda apresentada, acreditamos que a causa é nobre, no entanto, esse gasto não está relacionado à realização do evento; por isso, ultrapassa a possibilidade de uso, conforme apresentado no [Art. 2 do Estatuto desta Associação](https://github.com/apyb/estatuto).

O dinheiro excedente do caixa de todos os eventos que usam a estrutura da APyB é redirecionado da seguinte forma: 50% destinado para o [Fundo para Inclusão](https://apyb.python.org.br/apyb/fundo-inclus%C3%A3o) e 50% para manter a estrutura utilizada pelos eventos. 

O [Fundo para Inclusão](https://apyb.python.org.br/apyb/fundo-inclus%C3%A3o) é uma iniciativa aprovada durante a [Assembleia Geral de 2021](https://docs.google.com/document/d/1lz56OI3TLKvJdfjyC6RARnJZwZwYHmoqvX8GCrCMm1M/edit?usp=sharing), sendo possível pelos esforços da organização da Python Brasil daquele ano que, devido ao contexto econômico do alto valor arrecadado dos patrocínios, a comunidade viu a possibilidade de usar esse valor arrecadado para criação de um fundo para dar suporte à iniciativa cujo objetivo seja promover ação de diversidade e inclusão nas edições presenciais do evento. Sendo assim, a sobrevivência desse Fundo só é possível se cada evento tiver caixa excedente. 

A saber, esta foi a [ação promovida para a Python Brasil 2022](https://2022.pythonbrasil.org.br/programa-auxilio-financeiro), o qual a prestação de contas foi apresentada nesta [palestra relâmpago](https://www.youtube.com/live/6SShnyYTCv0?si=eKITpfvLNnWIdSSa&t=288).

Os cenários econômicos desde 2022 não permitiram acúmulo suficiente de verba para viabilizar uma nova ação com o Fundo para Inclusão. A Diretoria está esperançosa que com todos os eventos regionais acontecendo esse ano com excelente organização, como foi a Python Norte, consigamos realizar uma nova ação para Python Brasil 2025.

Acreditamos que seria enriquecedor um encontro aberto para trocas de experiências entre as diversas pessoas organizadoras dos eventos regionais de 2024, então estamos planejando este evento online em breve. 

@apyb/diretoria

**Reply by jailsonpj** [reply]
Mas essa iniciativa pode entrar como marketing relacionado ao evento. Tendo em vista que o dinheiro arrecado não estaria prejudicando a Python Brasil e nem muito menos a Python Norte que já foi realizado. 

Assegurando que isso entra como uma forma de marketing , a comunidade Python Norte pode usar isso como forma de divulgar os grupos que existem na região que não são de forma alguma representados na comunidade mãe (Python Brasil) . 

Levando os Kahunas a Python Brasil , conseguimos mostrar as outras comunidades os vários trabalhos que Pyladies Manaus , projeto Jyboia , Pydata Manaus , Grupy Am e PyNorte vem desenvolvendo na região .

Acredito que esse artigo citado assegura isso.

**Reply by anadulce** [reply]
Endosso o ponto do @jailsonpj , o estatuto é bastante vago, e também acredito que poderia ser interpretado facilmente como Divulgação de caso de sucesso ou marketing.
A Python Norte teve números de diversidade ímpares, temos que ter pessoas divulgando todo o contexto em que essa região está inserida para motivar tais acertos, deveria ser uma referência para toda a comunidade.
Além disso, as organizações têm autonomia para o gasto da verba arrecadada, se eles acreditam que ter a presença dos big kahunas atuais no evento é importante (tanto para divulgar o trabalho que foi feito, como para buscar potenciais apoios para as próximas edições), isso parece completamente alinhado ao escopo do evento, ainda é trabalho relativo ao evento.
Historicamente, as organizações promovem uma confraternização da organização usando parte das verbas remanescentes, não enxergo como a representação perante o evento nacional teria menos relevância que isso.

**Reply by talitarossari** [reply]
Estou de acordo com @anadulce e @jailsonpj.
O motivo da Python Brasil 2022 acontecer em Manaus foi aproximar as comunidades e tornar-nos mais diversos, além de aprender com a experiência da comunidade do Norte, que é ativa há anos.
Mesmo que tenha ocorrido uma Python Brasil na região num passado recente, não estou segura que conseguimos alcançar isso de forma realista, pois temos uma baixa adesão das comunidades Nortistas na comunidade Python. 
Como neste ano eles alcançaram uma marca de diversidade que buscamos há anos, praticamente [igualaram a presença de homens e mulheres no evento](https://docs.google.com/presentation/d/1fqDcQsHkhTSE01tj1zMt665XlWceiZq-gPnOHFWnbhE/edit#slide=id.g2d39d7db49c_0_36), acredito que podemos sim enquadrar esse gasto como Marketing e Divulgação de Cases de Sucesso.
Entendo o receio de que isso poderia abrir precedentes de uso do caixa do evento em prol de benefícios particulares, mas gostaria de ressaltar que não é este o intuito desta proposta. Aproximar as comunidades e criar conexões é importante tanto para que a Python Norte continue crescendo e abrangendo mais pessoas, tanto para que nós possamos aprender com o êxito que alcançaram em seus eventos. Estamos sugerindo que seja enquadrado como um gasto do evento, usando caixa remanescente, para que seja tratado com esta responsabilidade.

**Reply by rougeth** [reply]
> Estamos sugerindo que seja enquadrado como um gasto do evento, usando caixa remanescente, para que seja tratado com esta responsabilidade.

Avaliando especificamente a proposta acima, eu acho que esse não é um gasto válido pela organização do evento. A APyB considera que todo dinheiro arrecadado por um evento, deve ser usado **apenas** para realização desse evento. A Python Norte 2024 ter gastos com a Python Norte 2025 vai contra esse entendimento.

Um exemplo dessa regra sendo aplicada atualmente é o fato da APyB não permitir que o caixa remanescente dos eventos apoiados por ela seja repassados as suas próximas edições.

Dois pontos importantes que motivaram essa regra:
- É de responsabilidade das pessoas Kahunas arrecadar toda verba necessária para a realização do evento que estão organizando. Dessa forma a APyB garante que os eventos apoiados por ela são auto suficientes do ponto de vista financeiro.
- A verba arrecadada deve ser usada para fazer o melhor evento possível. Ou seja, não é um bom indicativo que os eventos não terminem com seus caixas **zerados**, tanto negativo (APyB tendo que cobrir os gastos do evento) quanto positivo (evento que não conseguiu usar todos os seus recursos para deixar o seu evento ainda melhor).

**Reply by jailsonpj** [reply]
@rougeth isso se restringe somente aos eventos regionais? Caso sobre receita na Python Brasil de 2024 , eu não posso usar na de 2025?

**Reply by rougeth** [reply]
> @rougeth isso se restringe somente aos eventos regionais?

Não. A instrução aos Kahunas da Python Brasil é a mesma, use todos os recursos arrecadados para o evento. Caso o evento termine com caixa positivo, como a Ju disse, 50% vai para o Fundo de Inclusão e 50% vai para o caixa da APyB.

A principal diferença entre os regionais e a Python Brasil, é que a Python Brasil é de responsabilidade da APyB, ou seja, em **caráter de emergência**, a APyB poderá usar de seus recursos para custear despesas do evento.
